{"name": "risk user -- @1.3 cases", "status": "passed", "steps": [{"name": "Given risk makes get request: /reconciliation/all_users_data/", "status": "passed", "start": 1610701619913, "stop": 1610701620053}, {"name": "Then check response of request: You do not have permission to perform this action. is False", "status": "passed", "start": 1610701620054, "stop": 1610701620054}], "parameters": [{"name": "url", "value": "/reconciliation/all_users_data/"}, {"name": "method", "value": "get"}, {"name": "bool", "value": "False"}], "start": 1610701619912, "stop": 1610701620054, "uuid": "e7399b99-ec3e-4a89-b0a3-41a43a733cd0", "historyId": "63af3909d8b41dbfee6ab9c349feb6b2", "fullName": "features/III-reconciliation_access.feature:risk user", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "reconciliation check access"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}